### ISSUE TITLE
pg_12_12: pg_dump failed regression

### ISSUE THREAD

**Issue by emotionbug** [bitnine-oss/agensgraph#561]
```
pg_dump: warning: archive items not in correct section order
pg_dump: error: query failed: ERROR:  syntax error at or near "regression"
LINE 1: ...CT oid FROM pg_database WHERE datname = ' dbname='regression...
                                                             ^
pg_dump: error: query was: SELECT setconfig[1] FROM pg_db_role_setting WHERE setrole = 0 AND setdatabase = (SELECT oid FROM pg_database WHERE datname = ' dbname='regression./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ'');
pg_dumpall: error: pg_dump failed on database "regression./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ", exiting
waiting for server to shut down.... done
server stopped
pg_dumpall of pre-upgrade database cluster failed
make[1]: *** [Makefile:48: check] 오류 1
make[1]: 디렉터리 '/home/emotionbug/CLionProjects/agensgraph-emotionbug/src/bin/pg_upgrade' 나감
make: *** [Makefile:43: check-pg_upgrade-recurse] 오류 2
```

**Comment by kysmou** [comment]
The pg_dump error is currently being fixed.

**Comment by ibrarahmad** [comment]
By the I am working on that issue.

**Comment by emotionbug** [comment]
Will be fixed in v2.5.1, v2.12.1.

https://github.com/bitnine-oss/agensgraph/commit/42aab446ca7a761f5cf9d99b67f612d629d79b74

**Comment by emotionbug** [comment]
The task is complete and will be included in the next release.
